modules/ocl-loader/src/main/java/org/eclipse/xpanse/modules/ocl/loader/data/models/Vm.java

```diff
@@ -17,13 +21,34 @@
 @EqualsAndHashCode(callSuper = true)
 public class Vm extends RuntimeBase {
 
+    @Length(min = 3, max = 64)
+    @Schema(description = "The name of the VM")
     private String name;
+
+    @NotBlank
```

Comment:
For these all string fields in the data model which are mandatory - let us add 3 validations always - NotNull, NotEmpty and NotBlank.